code/game/objects/structures/window.dm

```diff
@@ -393,20 +392,33 @@
 		if(!src) return
 		if(!fulltile)
 			return
-		var/junction = 0 //will be used to determine from which side the window is connected to other windows
+		/*var/junction = 0 //will be used to determine from which side the window is connected to other windows
 		if(anchored)
 			for(var/obj/structure/window/W in orange(src,1))
 				if(W.anchored && W.density	&& W.fulltile) //Only counts anchored, not-destroyed fill-tile windows.
 					if(src.wtype == W.wtype)
 						if(abs(x-W.x)-abs(y-W.y) ) 		//doesn't count windows, placed diagonally to src
 							junction |= get_dir(src,W)
-		icon_state = "[initial(icon_state)][junction]"
+		icon_state = "[initial(icon_state)][junction]"*/
 
-		overlays.Cut()
 		var/ratio = health / maxhealth
 		ratio = Ceiling(ratio*4) * 25
-		overlays += "damage[ratio]"
-		return
+
+		if(smoother)
+			smoother.smooth()
+			sleep(2)

```

Comment:
I suspect it was related to the singulo causing lots of resmoothing that would then be destroyed the next second